src/scrubbers.ts

```diff
@@ -596,3 +600,8 @@ export const defaultScrubbersSQL: ScrubbersSQLMap = {
   saltedHashSubstringScrubber: saltedHashSubstringScrubberSQL,
   keepCharsFromLeftScrubber: keepCharsFromLeftScrubberSQL,
 }
+
+const wrapIfMatchSQL = (ifMatch: string | undefined, expression: string): string => {
+  if (ifMatch === undefined) return expression
+  return `CASE WHEN LIKE(${sqlValueToReplace}, '${ifMatch}') THEN ${expression} ELSE ${sqlValueToReplace} END`
```

Comment:
```suggestion
  return `CASE WHEN REGEXP_LIKE(${sqlValueToReplace}, '${ifMatch}') THEN ${expression} ELSE ${sqlValueToReplace} END`
```
`LIKE` uses a different syntax (`%` instead of `.*`, etc, so better to use `REGEXP_LIKE`)